terraform/grafana.tf

```diff
@@ -0,0 +1,116 @@
+resource "aws_cloudwatch_log_group" "task" {
+  name = "platform/grafana"
+}
+
+data "template_file" "definition" {
+  template = "${file("grafana.json.template")}"
+
+  vars {
+    log_group_region = "${var.aws_region}"
+    log_group_name   = "${aws_cloudwatch_log_group.task.name}"
+  }
+}
+
+resource "aws_ecs_task_definition" "task" {
+  family                = "grafana_task_definition"
+  container_definitions = "${data.template_file.definition.rendered}"
+  task_role_arn         = "${module.ecs_grafana_iam.task_role_arn}"
+
+  volume {
+    name = "grafana"
+    host_path = "/ecs/grafana"
```

Comment:
Yeah, I don't think it should use the id_minter database. Creating another one just for this seems unreasonably expensive though... Would it be so bad to mount an EFS volume? It seems the cheapest option although not the easiest. 
Alternatively we could spin up a t2.nano or t2.small ec2 instance with mysql for this. It's probably cheaper than aurora?